src/main/java/com/google/devtools/build/lib/packages/RepoFileGlobals.java

```diff
@@ -16,44 +16,62 @@
 
 import java.util.Map;
 import net.starlark.java.annot.Param;
+import net.starlark.java.annot.ParamType;
 import net.starlark.java.annot.StarlarkMethod;
 import net.starlark.java.eval.EvalException;
+import net.starlark.java.eval.Sequence;
 import net.starlark.java.eval.Starlark;
 import net.starlark.java.eval.StarlarkThread;
 
 /** Definition of the {@code repo()} function used in REPO.bazel files. */
-public final class RepoCallable {
-  private RepoCallable() {}
+public final class RepoFileGlobals {
+  private RepoFileGlobals() {}
 
-  public static final RepoCallable INSTANCE = new RepoCallable();
+  public static final RepoFileGlobals INSTANCE = new RepoFileGlobals();
+
+  @StarlarkMethod(
+      name = "ignore_directories",
+      useStarlarkThread = true,
+      documented = false, // TODO
```

Comment:
unTODO please :)

Reply:
actually, this file doesn't even show up in docs yet (https://bazel.build/rules/lib/globals). The reason was that the `repo()` function is actually special and was documented separately in the build encyclopedia (since its args are "dynamic"). I'll send a follow-up change later to add this to docs. (but please still write the docstring)

Reply:
Done!